services/kafka/config.go

```diff
@@ -49,7 +49,7 @@ type Config struct {
 }
 
 func NewConfig() Config {
-	return Config{ID: DefaultID}
+	return Config{ID: DefaultID, SASLAuth: SASLAuth{SASLOAUTHExpiryMargin: 10 * time.Second}}
```

Comment:
Please define the `SASLOAUTHExpiryMargin` value as a constant, similar to `DefaultID`.

```suggestion
	return Config{ID: DefaultID, SASLAuth: SASLAuth{SASLOAUTHExpiryMargin: DefaultSASLOAUTHExpiryMargin}}
```

Reply:
Thanks, fixed